app/views/posts/index.html.erb

```diff
@@ -1,6 +1,21 @@
-<h1>List of all posts</h1>
-<ul>
-  <li>post 1</li>
-  <li>post 2</li>
-  <li>post 3</li>
-</ul>
+<div style="border: #2E2F30">
+  <h1><%= @user.name %></h1>
+  <h4>Number of posts<%= @posts.count %></h4>
+</div>
+
+<div>
+  <% @posts.each do |post| %>
+    <div>
+      <div>
+        <h4>Post #<%= post.id %></h4>
+        <p><%=post.text %></p>
+        <p>Comments: <%=post.comments.count %>, Likes: <%= post.likes.count %></p>
+      </div>
+      <div>
+        <% post.comments.each do |comment| %>
+          <p><%= comment.user.name %>: <%= comment.text %></p>
+        <% end %>
+      </div>
+    </div>
+  <% end %>
+</div>
```

Comment:
| @Alibaba2023 Post Index | Microverse Post Index |
|-|-|
| ![image](https://github.com/Alibaba2023/blog_app/assets/93039359/311363e3-3d69-4731-8519-eaaf0aabc812) | ![image](https://github.com/Alibaba2023/blog_app/assets/93039359/e0999fc5-9653-4cbf-8cf0-5f26d9c3f0e2) |

- [ ] It would be great if you follow the given wireframe to make your design look professional and attractive 👍
- [ ] It would be great if you use methods that you have created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md). to follow the project requirements 😊